code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -253,6 +253,9 @@
 			return
 		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
 						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
+	else if(check_zone(M.zone_selected) == "head") //Headpats!
+		M.visible_message("<span class='notice'>[M] gives [src] a pat on the head to make [p_them()] feel better!</span>", \

```

Comment:
M.visible_message("<span class='notice'>[M] punches [src] in the eye to make [p_them()] feel better!</span>", \
"<span class='notice'>You punch [src] in the eye to make [p_them()] feel better!</span>")